### PR TITLE
optionally show previous / next links on post page

### PR DESCRIPTION
**Pull request by jon-walton**
### Description

For my blog, I prefer to not show the next / previous links. This PR adds a config item to conditionally show them

### CODE REVIEW

**Comment by nabobalis** [comment]
Thanks for the PR, can you also add it to https://github.com/sunpy/ablog/blob/main/ablog/start.py, I believe all the other options should be there even if they are commented out.

**Comment by jon-walton** [comment]
thanks @nabobalis , that's added 😄

File: ablog/start.py [hunk 1]
```diff
@@ -103,6 +103,11 @@ def w(t, ls=80):
 # is ``False``.
 # post_always_section = False
 
+# When ``True``, links to the previous and next posts will be rendered at the
```
Comment (nabobalis) [inline ablog/start.py:106]:
```suggestion
# When ``True``, links to the previous and next posts will be rendered at the
```

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Thanks for the PR @jon-walton

**Pull request merged**